OnBaseDocsApi/Controllers/OnBaseDocsController.cs

```diff
@@ -207,26 +208,57 @@ public async Task<IHttpActionResult> Post()
         public IHttpActionResult ListDocs()
         {
             var parms = new ParamCollection(Request.RequestUri.ParseQueryString());
-            var indexKey = parms.Get("filter[indexKey]", DefaultIndexKey);
-            var typeGroup = parms.Get("filter[typeGroup]", DefaultTypeGroup);
-            var startDocId = parms.Get("filter[startDocId]", DefaultStartDocId);
-            var pageSize = parms.Get("filter[pageSize]", DefaultPageSize);
+            var filterIndexKey = parms.Get("filter[indexKey]", DefaultIndexKey);
+            var filterDocTypeGroup = parms.Get("filter[typeGroup]", DefaultTypeGroup);
+            var filterDocType = parms.Get("filter[type]", DefaultDocType);
+            var filterStartDocId = parms.Get("filter[startDocId]", DefaultStartDocId);
+            var filterPageSize = parms.Get("filter[pageSize]", DefaultPageSize);
+
             // We bundle the bad requests in one response.
             var badRequestErrors = new List<Error>();
 
-            if (string.IsNullOrWhiteSpace(indexKey))
-                badRequestErrors.Add(BadRequestError("The required parameter filter[indexKey] is missing."));
+            //
+            // Read the keywords.
+            //
+            var filterKeywords = new Dictionary<string, string>();
+            var keywordsParam = parms.Get("filter[keywords][hasAll]", string.Empty);
```

Comment:
I think this should be consistent with the other parameters and have a `DefaultKeywords` var